setup-homebrew/README.md

```diff
@@ -27,3 +27,13 @@ This also sets up the variables necessary to cache the gems installed by Homebre
   if: steps.cache.outputs.cache-hit != 'true'
   run: brew install-bundler-gems
 ```
+
+If you're using Linux, it will be quicker to use the GitHub Actions provided Ruby:
```

Comment:
Why is that? I thought we supply portable Ruby.

Reply:
We do but it's way quicker to use the (already installed) version in GitHub Actions which requires only a symlink change.

Reply:
You are saying that our portable Ruby is somehow slower than the one provided by GHA?

Reply:
I'm saying that installing it is: yes.

Reply:
But AFAIK portable Ruby is already installed with Homebrew in GHA VM images, so the cost of installing it on runtime is zero.

Reply:
Last time I checked it wasn't but I could well be wrong here?

Reply:
It is installed, there is no indication in logs that would tell it is being installed each time a workflow job is run and in `homebrew-core` we do not setup Ruby anywhere now.

Reply:
See this test workflow run:
https://github.com/dawidd6/homebrew-test/runs/952878355?check_suite_focus=true